### ISSUE TITLE
spock-parent has a dependency on org.codehaus.groovy:groovy-all:1.8.0-beta-3-SNAPSHOT

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sampig/spock#213]
```
Given this build.gradle file:

apply plugin: 'groovy'
apply plugin: 'eclipse'

sourceCompatibility = 1.6
version = '1.0.0-SNAPSHOT'

repositories {
    mavenCentral()
}

dependencies {
    groovy group: 'org.codehaus.groovy', name: 'groovy', version: '1.8.3'
    testCompile "org.spockframework:spock-core:0.5-groovy-1.8"
    testCompile group: 'junit', name: 'junit', version: '4.10'
}


When I run gradle test,


Then I get this error:
* What went wrong:
Could not resolve all dependencies for configuration ':testCompile':
    - unresolved dependency: org.codehaus.groovy#groovy-all;1.8.0-beta-3-SNAPSHOT: not found
Cause: not found


```

Original issue reported on code.google.com by `time...@gmail.com` on 2 Nov 2011 at 6:44

**Comment by GoogleCodeExporter** [comment]
```
Excluding the transitive groovy-all dependency is a workaround for the 
unresolvable SNAPSHOT, but still I am not fond of this solution:

dependencies {
    groovy group: 'org.codehaus.groovy', name: 'groovy', version: '1.8.3'
    testCompile("org.spockframework:spock-core:0.5-groovy-1.8") {
        exclude module: 'groovy-all'
    }
    testCompile group: 'junit', name: 'junit', version: '4.10'
}
```

Original comment by `time...@gmail.com` on 2 Nov 2011 at 6:50

**Comment by GoogleCodeExporter** [comment]
```
At the time Spock 0.5 was released, Groovy 1.8-beta-3 was the latest Groovy 1.8 
version. There are known issues with Spock 0.5 and Groovy 1.8 final. I 
recommend to use Spock 0.6-groovy-1.8-SNAPSHOT until Spock 0.6 gets released 
(which should happen soon).
```

Original comment by `pnied...@gmail.com` on 2 Nov 2011 at 6:56
- Changed state: **WontFix**

**Comment by GoogleCodeExporter** [comment]
```
That worked. Thanks.
```

Original comment by `time...@gmail.com` on 2 Nov 2011 at 7:36
